packages/next/src/build/webpack/plugins/define-env-plugin.ts

```diff
@@ -14,16 +18,16 @@ function errorIfEnvConflicted(config: NextConfigComplete, key: string) {
   }
 }
 
+type BloomFilter = ReturnType<
+  import('../../../shared/lib/bloom-filter').BloomFilter['export']
```

Comment:
Why not import at the top?

Reply:
`export` is a method of `BloomFilter`, theoretically we could import `BloomFilter` at the top but didn't want to make too many changes, kept it as-is from this line: https://github.com/vercel/next.js/pull/63128/files#diff-28c0329619b90836564a7769c43ff473dbd28e8e31b75e650624da4d06911763L22 just sharing the type more.